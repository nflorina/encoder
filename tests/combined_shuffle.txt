aaaaa
88374
c
13
END
