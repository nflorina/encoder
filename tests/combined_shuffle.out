1 1 1
aaaaa88374ac3
c3a3a4a7a8a8a
